MAINTAINERS.md

```diff
@@ -1,9 +1,11 @@
 # Lima maintainers
 
-| Name         | GitHub ID (not Twitter ID)                     | E-mail address for reporting security issues | GPG fingerprint                                                                          |
-|--------------|------------------------------------------------|----------------------------------------------|------------------------------------------------------------------------------------------|
-| Akihiro Suda | [@AkihiroSuda](https://github.com/AkihiroSuda) | (See GitHub profile)                         | [C020 EA87 6CE4 E06C 7AB9  5AEF 4952 4C6F 9F63 8F1A](https://github.com/AkihiroSuda.gpg) |
-| Jan Dubois   | [@jandubois](https://github.com/jandubois)     | (See GitHub profile)                         |                                                                                          |
+| Name               | GitHub ID (not Twitter ID)                     | E-mail address for reporting security issues | GPG fingerprint                                                                          |
+|--------------------|------------------------------------------------|----------------------------------------------|------------------------------------------------------------------------------------------|
+| Akihiro Suda       | [@AkihiroSuda](https://github.com/AkihiroSuda) | (See GitHub profile)                         | [C020 EA87 6CE4 E06C 7AB9  5AEF 4952 4C6F 9F63 8F1A](https://github.com/AkihiroSuda.gpg) |
+| Jan Dubois         | [@jandubois](https://github.com/jandubois)     | (See GitHub profile)                         |                                                                                          |
+| Anders Björklund   | [@afbjorklund](https://github.com/afbjorklund) | (See GitHub profile)                         |                                                                                          |
```

Comment:
Once upon a time I started to use my middle initial... Might as well do it here too, for consistency ?

Anders F Björklund

Reply:
Updated